### PR TITLE
Details popup window- 1/6/2023

### PR DESCRIPTION
**Pull request by fithamlak**
Details popup window all requirements are implemented
 =>You should implement the following interactions:
    -When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
    -When the user clicks (or taps) the close (X) button, the popup disappears.
=> In order to associate each project with the popup window details, you must refactor the project section:
  -You need to use a JavaScript array to store all of the information for all projects.
  -For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, 
     featured image, technologies, link to live version, and link to the source.
 -You need to update the main page so that the projects section is created dynamically using the information stored in that 
   JavaScript object. Remember, all of the HTML in that section is created when the page loads.
 -After that, you could implement the popup window.

### CODE REVIEW

**Comment by huseyinbiyik** [comment]
Hi @fithamlak and @Ellyboi 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

❌ You have submitted a project, that has no ESLint JavaScript linter set up. You should have set it up on the previous project. Kindly follow this [link](https://github.com/microverseinc/linters-config/tree/master/html-css-js) and set up ESLint. Also please solve the git conflict before submit the project for the review. 🦾

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.js [hunk 1]
```diff
@@ -23,7 +34,183 @@ function closeNav() {
...
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    technologies: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstraps'],
+    ImageForMobile: '/images/fourth-work.svg',
+    ImageForDesktop: '/images/work4-desktop-view.svg',
+    accomplishmentDetailModel: '/images/accomplishment-details-model.svg',
+    alt: 'work_multi-post_stories_daily_personalized_reads',
+    live: 'https://fithamlak.github.io/',
+    source: 'https://github.com/fithamlak/My-portfolio',
+  },
+];
```
Comment (AshakaE) [inline index.js:100]: - [ ] Kindly work on the image path, they appear broken
- 
image

**Review by AshakaE — CHANGES_REQUESTED** [review]
Hi @fithamlak 👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Pull request 👍 
- Git flow ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js [hunk 2]
```diff
@@ -23,7 +34,183 @@ function closeNav() {
...
+    ImageForDesktop: './images/work4-desktop-view.svg',
+    accomplishmentDetailModel: './images/accomplishment-details-model.svg',
+    alt: 'work_multi-post_stories_daily_personalized_reads',
+    live: 'https://fithamlak.github.io/',
+    source: 'https://github.com/fithamlak/My-portfolio',
+  },
+];
+
+// function to display details popup windo when seeProjrct buttons of each accomplishment clicked
+function seeProject(k) {
```
Comment (Ismailco) [inline index.js:103]: - [ ] [OPTIONAL] Kindly make sure the right image is displayed on the popup, as for now only one image is displayed on all the popups

**Review by Ismailco — APPROVED** [review]
### Hi @fithamlak, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 2 above)
Comment (fithamlak) [reply index.js:103]: Yes,  I intentionally display one image  for desktop version, and its own different image for the mobile version

**Pull request closed without merging**